### DISCUSSION TITLE
Using old Thinkpad Back/Prev and Forward/Next keys to toggle workspaces

### DISCUSSION THREAD

**Discussion by omaralt** [i3/i3#4779] [Show and tell]
Hi first post on here. I hope you are all having a good new year. I was trying to bind the old back and forward keys directly above the left and right keys of a Thinkpad x220 to cycle back and forward across Workspaces. 

```
#bindcode to use thinkpad next and previous track keys to workspace next and prev
bindcode $mod+167 workspace next
bindcode $mod+166 workspace prev
```

```
#bind < and > less and greater than if not using thinkpad
bindcode $mod+Shift+60 workspace next
bindcode $mod+Shift+59 workspace prev
```

**Comment by Airblader** [comment]
What's your question?

Also, your bindings are unnecessarily convoluted. There's no need to route the command through exec + i3.

**Reply by omaralt** [reply]
Apologies I put it in the show and tell
